main.go

```diff
@@ -134,3 +138,14 @@ func getYear() string {
 	}
 	return strconv.Itoa(time.Now().Year())
 }
+
+func getProject() string {
+	if fProject != "" {
+		return fProject
+	}
+	rootDir, err := filepath.Abs(filepath.Dir(os.Args[0]))
```

Comment:
The updated documentation in the README says:
> The current folder is used as the project name if `-project` is omitted.

Based on this, shouldn't this code use `os.Getwd`, not `os.Args[0]`?